docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.less

```diff
@@ -12,4 +12,8 @@
         text-transform: none;
         padding: 4px 12px;
     }
-}
\ No newline at end of file
+}
+
+.hpe-icon-instructions {
+    margin-bottom: 5px;
+}
```

Comment:
Should be okay to remove this. Just use the same styles that the ux-icon documentation uses.

Reply:
I have removed this but I had to add a margin underneath the `ux-alert` that replaced it.